src/protocols/aeternity/views/TransferSendModal.vue

```diff
@@ -5,6 +5,7 @@
     :hide-arrow-send-icon="isMultisig"
     :custom-primary-button-text="customPrimaryButtonText"
     :sending-disabled="isSendingDisabled"
+    :is-air-gap="isAirGap"
```

Comment:
As there might be more account types (like for example ledger and so on) I suggest to try avoid using props related directly to this account type. Maybe passing the active account as a prop and trying to establish the component's logic by analysing it would be better?

Reply:
From your previous comments I've already removed the `isAirGap` prop and used the `accounts` composable to get the `isActiveAccountAirGap` ref in order to do the required checks in the base component